sys/include/net/gnrc/ipv6/nib/abr.h

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup
+ * @ingroup
+ * @brief
+ * @{
+ *
+ * @file
+ * @brief
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef NET_GNRC_IPV6_NIB_ABR_H
+#define NET_GNRC_IPV6_NIB_ABR_H
+
+#include "net/ipv6/addr.h"
+#include "net/gnrc/ipv6/nib/conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#if     (GNRC_IPV6_NIB_CONF_6LBR && GNRC_IPV6_NIB_CONF_MULTIHOP_P6C) || defined(DOXYGEN)
```

Comment:
too many white spaces for my taste :P

Reply:
```C
#if(GNRC_IPV6_NIB_CONF_6LBR&&GNRC_IPV6_NIB_CONF_MULTIHOP_P6C)||defined(DOXYGEN)
```

better?

![](https://static.comicvine.com/uploads/original/11115/111153702/3860689-1090dc1fdc42e58d45dc1a374db6fbdb2292d5fcaba942aeddec8558b50a70b6.jpg)

Reply:
(just joking, will fix)